spread.yaml

```diff
@@ -11,6 +11,35 @@ exclude:
   - .github
 
 backends:
+  # We can switch back to using the official Spread LXD backend after
+  # https://github.com/snapcore/spread/pulls?q=is%3Apr+is%3Aopen+LXD+updated%3A%3C%3D2024-02-23+
+  lxd:
+    type: adhoc
+    allocate: |
+      set -x
+      release=$(echo $SPREAD_SYSTEM | awk -F '-' '{print $2}')
+
+      # Ideally, we would add the ubuntu-minimal remote
+      # e.g. https://cloud-images.ubuntu.com/minimal/releases/jammy/
+      # but that would effectively change the host's LXC configurations.
+
+      echo "Allocating $SPREAD_SYSTEM..."
+      lxc launch --ephemeral ubuntu:$release $SPREAD_SYSTEM
+      until lxc exec $SPREAD_SYSTEM -- pgrep sshd &>/dev/null
+      do
+        sleep 5
+      done
```

Comment:
```suggestion
      until lxc exec $SPREAD_SYSTEM -- systemctl status | grep "running"
      do
        sleep 5
      done
      lxc exec $SPREAD_SYSTEM -- systemctl enable --now ssh
```

Reply:
This made it work for me

Reply:
Hum, that seems redundant but harmless...giving it a try